packages/mock-app/src/components/GenericFeature/GenericFeature.tsx

```diff
@@ -141,7 +141,17 @@ export const GenericFeature: React.FC<IGenericFeatureProps> = ({ components, ser
             break;
           case ComponentType.Submit:
             props.onClick = async () => {
-              await executeServices(services, state);
+              try {
+                const result = await executeServices(services, state);
+                if (!result)  {
+                  return toastMessage({ status: Status.error, message: 'Something went wrong' });  
+                }
+
+                toastMessage({ status: Status.success, message: 'Submit success' });
```

Comment:
Should we hardcode the `Submit` message here? What if the button is Download or ScanQR button?